### PR TITLE
fix: ui tweaks for custom sender name

### LINKED ISSUE #1218: UI/UX fixes for Custom Sender Name
Amendments for Custom Sender Name:

1. Remove 'Postman.gov.sg' in the text field. 
**Why:** User should see the placeholder and via append immediately to understand the feature more intuitively. We also want to discourage the use of 'Postman.gov.sg' cause that's the issue we're trying to solve.
Screenshot 2021-06-02 at 12 52 06 PM

2. Remove full stop for 'Sender details'. Generally, when an instruction or description is composed of a single sentence, we do not include a period at the end of the sentence. When there is more than one sentence, all sentences should have a period. Can read more about it's practised for this team [here](https://docs.appian.com/suite/help/19.2/ux_formatting_and_punctuation.html#:~:text=Period%20Usage,sentences%20should%20have%20a%20period.)

3. For all mentions of 'e.g.' - make them lowercase. Meaning capital 'E' should change to lowercase 'e'
Screenshot 2021-06-02 at 1 36 03 PM

4. Description changes:
**Subject**
Keep it short, specific and personalised. Try to use less than 10 words.

5. Change default sender email to 'donotreply@postman.gov.sg' as discussed

### PR DESCRIPTION
**Pull request by lamkeewei**
## Problem

Closes #1218 

## Solution

**Improvements**:
- Updated copy for input placeholders
- Do not pre-populate default sender name
- Switch default mail from address on backend config to `donotreply@postman.gov.sg`

## Tests

**Sender name should not be pre-populated**
- Create a new email campaign with default from address
- Observe that sender name is not populated

**Custom sender name for default from address**
- Create a new email campaign with default from address
- Provide a value for the sender name and proceed to save template
- Navigate back to the template page and observe that sender name is populated with the custom value provided in the previous step
- Send campaign. Test and campaign emails should have the correct from name and address

**Using `donotreply@postman.gov.sg`**
- Set `BACKEND_SES_FROM` and `WORKER_SES_FROM` to use `donotreply@postman.gov.sg` in either your env file or Secrets Manager
- Create and send an email campaign
- Observe that new mail from should be using `donotreply@postman.gov.sg`
- Check that messages are marked as success in the delivery report to ensure email delivery notifications are working correctly

**Backward compatibility for `donotreply@mail.postman.gov.sg`**
- Set `BACKEND_SES_FROM` and `WORKER_SES_FROM` to use `donotreply@postman.gov.sg` in either your env file or Secrets Manager
- Duplicate a previous campaign using `donotreply@mail.postman.gov.sg`
- Campaign should be able to be sent out.
- Send transactional email with `donotreply@mail.postman.gov.sg` and ensure that it is accepted.

## Deploy notes
Update the following environment variables in Secrets Manager
- `BACKEND_SES_FROM` - `Postman <donotreply@postman.gov.sg>`
- `WORKER_SES_FROM` - `Postman <donotreply@postman.gov.sg>`

### CODE REVIEW

File: backend/src/core/config.ts [hunk 1]
```diff
@@ -459,9 +459,7 @@ const config = convict({
 })
 
 // If mailFrom was not set in an env var, set it using the app_name
-const defaultMailFrom = `${config.get(
-  'APP_NAME'
-)} <donotreply@mail.postman.gov.sg>`
+const defaultMailFrom = `${config.get('APP_NAME')} <donotreply@postman.gov.sg>`
```
Comment (mjgui) [inline backend/src/core/config.ts:462]: Do we need to change this for `worker` and `serverless` too?

File: frontend/src/components/dashboard/create/email/EmailTemplate.tsx [hunk 2]
```diff
@@ -179,15 +178,26 @@ const EmailTemplate = ({
...
       setFromAddress(selectedFromAddress)
+
+      // Populate from name only for custom from address or if a from name
+      // has already been set.
+      if (
+        !selectedFromAddress.endsWith(t`defaultEmailDomain`) ||
+        initialFromName
+      ) {
+        // Use custom from name if it has already been set. For e.g.,
+        // for "Custom <donotreply@mail.postman.gov.sg>"", we should
```
Comment (mjgui) [inline frontend/src/components/dashboard/create/email/EmailTemplate.tsx:190]: Just a minor thing, should be `@postman.gov.sg`

File: backend/src/core/config.ts (hunk 1 above)
Comment (lamkeewei) [reply backend/src/core/config.ts:462]: Ah yes. Good catch!

**Comment by lamkeewei** [comment]
Closing and will reopen to only include ui fixes (no mail from changes)

**Pull request closed without merging**